SPEC.md

```diff
@@ -539,6 +542,35 @@ The frame bookend is a simple event that can be used to determine that the entir
 | 0x1 | Frame Number | int32 | The number of the frame. Starts at -123. Frame 0 is when the timer starts counting down | 3.0.0
 | 0x5 | Latest Finalized Frame | int32 | For non-rollback, this should always equal the frame number. For rollback, this indicates the index of a frame which is guaranteed not to happen again (finalized). This is very useful when reading a file in real-time to make sure you are processing "finalized" information | 3.7.0
 
+### FOD Platforms
+This event only occurs on Fountain of Dreams, and is sent every frame the height of the platform is changing.
```

Comment:
It is sent for each platform every frame the height changes. That means you can get both platforms changing and get two events in the same frame.